pkg/jx/cmd/controller_team.go

```diff
@@ -306,6 +306,11 @@ func (o *ControllerTeamOptions) onTeamChange(obj interface{}, kubeClient kuberne
 			}
 		}
 
+		err = kube.SetAdminNamespace(kubeClient, teamNs, adminNs)
+		if err != nil {
+			log.Errorf("Unable set admin namespace on team %s to %s - %s", util.ColorInfo(teamNs), util.ColorInfo(adminNs), err)
```

Comment:
error printing should be `%v`

Reply:
looks like they are equivalent for errors

Reply:
it appears to make no difference and I was copying the rest of the code.

```
	log.Errorf("this is a test error with percent s  : %s", err)
	log.Errorf("this is a test error with percent v  : %v", err)
	log.Errorf("this is a wrapped error with percent s  : %s", errors.Wrap(err, "a wrapped message"))
	log.Errorf("this is a wrapped error with percent v  : %v", errors.Wrap(err, "a wrapped message"))
```
produces
```
this is a test error with percent s  : Failed to obtain the team namespace (myteamBBBAD) when getting the admin namespace: namespaces "myteamBBBAD" not found
this is a test error with percent v  : Failed to obtain the team namespace (myteamBBBAD) when getting the admin namespace: namespaces "myteamBBBAD" not found
this is a wrapped error with percent s  : a wrapped message: Failed to obtain the team namespace (myteamBBBAD) when getting the admin namespace: namespaces "myteamBBBAD" not found
this is a wrapped error with percent v  : a wrapped message: Failed to obtain the team namespace (myteamBBBAD) when getting the admin namespace: namespaces "myteamBBBAD" not found
```